### ISSUE TITLE
small fixes in svnstats

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waterxi/svnplot#6]
```
The weekdays were incorrect and you did not consider timezones. Here are 
the small fixes required in svnstats.py

 1. basically all your strftime etc should use 'localtime' modifier
 2. weekdaylist.append(calendar.day_abbr[int(dayofweek)-1]) (sunday is 0 
not monday)

```

Original issue reported on code.google.com by `andre.ka...@gmail.com` on 17 May 2009 at 7:09

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the bug report and suggestions. I will check and add the corrections 
as per your suggestion. It will be available in next release.
```

Original comment by `nitinbh...@gmail.com` on 18 May 2009 at 4:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 18 May 2009 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.5.5
```

Original comment by `nitinbh...@gmail.com` on 19 May 2009 at 9:02
- Changed state: **Fixed**
